packages/morph/components/editor.tsx

```diff
@@ -139,31 +139,33 @@ export default function Editor({ vaultId }: EditorProps) {
   const fetchNewNotes = async (content: string) => {
     try {
       const apiEndpoint = process.env.NEXT_PUBLIC_API_ENDPOINT
-
+      
       // TODO: should disable or greyed out the notes button
       if (!apiEndpoint) throw new Error("ENDPOINT cannot be found, notes won't be functional")
 
-      return await axios
-        .post<AsteraceaResponse, AxiosResponse<AsteraceaResponse>, AsteraceaRequest>(
-          `${apiEndpoint}/suggests`,
-          {
-            essay: content,
-            num_suggestions: 8,
-            max_tokens: 8192,
-          },
-          {
-            headers: {
-              Accept: "text/event-stream",
-              "Content-Type": "application/json",
-            },
+      const resp = await axios.post<
```

Comment:
We probably need to check for toggling from keyboard events and onclick button 🤔